brave/brave-core
# Do not start process if rewards is off

Resolves https://github.com/brave/brave-browser/issues/9526

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- start browser
- open task manager
- make sure that there is no ledger process
- join rewards
- make sure that all looks good and you can claim promotions
- disable rewards 
- open task manager
- make sure that there is no ledger process
- toggle on/off and make sure that process is managed correctly

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
